docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,53 +1,73 @@
-++++
+== Add data
 
-To start working with your data in {kib}, use one of the many ingest options,
-available from the home page.
-You can collect data from an app or service,
-or upload a file that contains your data.  If you're not ready to use your own data,
-add a sample data set and give {kib} a test drive.
+The best way to add data to the Elastic Stack is to use one of our many integrations.
+Alternatively, you can add a sample data set or upload a file.
+
+[float]
+[[add-an-integration]]
+=== Add an integration
+
+Integrations are a way to add logs, metrics, traces,
+content, and more from your apps, endpoints, infrastructure, cloud, network,
+workplace tools, and other common sources in your ecosystem. All
+integrations are available in a single view, and
+{kib} guides you there from the Welcome screen and home page.
 
 [role="screenshot"]
-image::images/addData_home_7.15.0.png[Built-in options for adding data to Kibana:  Add data, Add Elastic Agent, Upload a file]
+image::images/add-integration.png[Integrations page from which you can choose integrations to start collecting and analyzing data]
+
+If an integration is available for both
+https://www.elastic.co/guide/en/fleet/master/beats-agent-comparison.html[Elastic Agent and Beats], the *Integrations* view defaults to the
+Elastic Agent integration, if it is generally available (GA).
+To switch to a
+Beats integration, use the filter on the bottom left.
 
 [float]
-[[add-data-tutorial-kibana]]
-=== Add data
+==== Secure your data
 
-Want to ingest logs, metrics, security, or application data?
-Install and configure a Beats data shipper or other module to periodically collect the data
-and send it to {es}. You can then use the pre-built dashboards to explore and analyze the data.
+Collect security events and threat intelligence with Filebeat,
+Auditbeat, Winlogbeat, Packetbeat, and Elastic Agent.
+Install the Endpoint Security integration for Elastic Agent to
+protect your hosts with threat detection and prevention.
 
-[role="screenshot"]
-image::images/add-data-tutorials.png[Add Data tutorials]
+[float]
+==== Observe your data
 
-[discrete]
-=== Add Elastic Agent
+Get logs, metrics, and uptime data into the Elastic Stack.
+Integrations are available for popular services and platforms,
+such as Nginx, AWS, and MongoDB,
+and generic input types like log files.
+Refer to {observability-guide}/observability-introduction.html[What is Elastic Observability] for more information.
+
+[float]
+==== Search your data
+Discover, extract, and index your web content into App Search engines using the
+{app-search-ref}/web-crawler.html[Enterprise Search web crawler].
+Search across Google Drive, GitHub, Salesforce, and many other web services using
+{workplace-search-ref}/workplace-search-content-sources.html[Workplace Search content sources].
 
-*Elastic Agent* is the next generation of
-data integration modules, offering
-a centralized way to set up your integrations.
+[float]
+==== Language clients
```

Comment:
The other section titles seem more focused on value. Thoughts on doing something like this? "Develop on your data", "Build on your data", "Ship any data", "Analyze custom data", etc.

Reply:
good idea.